src/billing/components/PayAsYouGo/CancellationOverlay.tsx

```diff
@@ -56,6 +56,7 @@ const CancellationOverlay: FC<Props> = ({onHideOverlay}) => {
       reason: VariableItems[reason],
       canContactForFeedback: canContactForFeedback ? 'true' : 'false',
     }
+    event('CancelServiceExecuted Event', payload)
```

Comment:
does this work? i dont think the event -> line protocol translator will handle nested objects like `suggestions`

Reply:
`suggestions` is a string. If can `JSON.stringify(payload)`, if you'd like!

Reply:
make sure you need that string and that you're not blowing up cardinality with putting this in a tag, but g2g other than that